### PR TITLE
Documentation corrections.

### PR DESCRIPTION
**Pull request by jlwalke2**

### CODE REVIEW

File: saspy/sasutil.py [hunk 1]
```diff
@@ -108,13 +105,12 @@ def hpsample(self, **kwargs: dict) -> 'SASresults':
 
         ``required_set = {}``
 
-        ``legal_set= {'cls', 'code', 'grow', 'id', 'model', 'out'
-        'partition', 'performance', 'prune', 'rules'}``
+        ``legal_set= { 'class', 'performance', 'target', 'var'}
```
Comment (FriedEgg) [inline saspy/sasutil.py]: class should be cls, this is the same for outside the documentation for the actual assignment of legal_set

File: saspy/sasutil.py [hunk 2]
```diff
@@ -61,22 +61,20 @@ def hpimpute(self, **kwargs: dict) -> 'SASresults':
...
         Python method to call the HPIMPUTE procedure
 
-        ``required_set = {}``
+        ``required_set = {'input'}``
 
-        ``legal_set= {'cls', 'code', 'grow', 'id', 'model', 'out'
-        'partition', 'performance', 'prune', 'rules'}``
+        ``legal_set= {'input', 'impute', 'performance', 'id', 'freq', 'code'}``
 
         cls is an alias for the class statement
```
Comment (FriedEgg) [inline saspy/sasutil.py]: this line is not relevant since HPIMPUTE does not have a class statement

File: saspy/sasutil.py [hunk 3]
```diff
@@ -86,12 +84,11 @@ def hpbin(self, **kwargs: dict) -> 'SASresults':
 
         ``required_set = {}``
 
-        ``legal_set= {'cls', 'code', 'grow', 'id', 'model', 'out', 'partition', 'performance', 'prune', 'rules'}``
+        ``legal_set= {'code', 'id', 'performance', 'target', 'input'}``
```
Comment (FriedEgg) [inline saspy/sasutil.py]: missing the FREQ statement

File: saspy/sasutil.py (hunk 1 above)
Comment (jld23) [inline saspy/sasutil.py]: Correct. since class is a python reserved word it is cls everywhere doc and code

**Comment by jld23** [comment]
@jlwalke2 thanks for submitting this. We'll review and address soon.

**Comment by jld23** [comment]
@jlwalke2 can you make the changes suggested by @FriedEgg and then update us so that we can commit your pull request?

**Comment by jlwalke2** [comment]
I've updated the request with the changes recommended by @FriedEgg.  Just let me know if there are any additional updates needed.

**Comment by tomweber-sas** [comment]
Jared, does this all look good now? If so, we can merge in this PR.
-Tom

**Comment by jld23** [comment]
Yes. it looks good to me.

**Comment by tomweber-sas** [comment]
PR merged. Thanks for contributing!
-Tom

**Pull request closed without merging**